web/Makefile

```diff
@@ -11,16 +11,14 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-include ../Makefile.INCLUDE
-
 all: blob/files.go
 
 SUFFIXES:
 
 blob/files.go: $(shell find templates/ static/ -type f)
-	./blob/embed-static.sh static templates | $(GOFMT) > $@
+	./blob/embed-static.sh static templates | gofmt > $@
```

Comment:
If we kept the `make web` step in the parent Makefile (to be executed manually), we could leave this as is, no?